src/main/java/com/metriql/warehouse/clickhouse/ClickhouseWarehouse.kt

```diff
@@ -29,7 +29,8 @@ object ClickhouseWarehouse : Warehouse<ClickhouseConfig> {
         val password: String,
         val ssl: Boolean?,
         val usePool: Boolean,
-        val connectionParameters: Map<String, String>?
+        val connectionParameters: Map<String, String>?,
+        val driverUriParameters: Map<String, String>?,
```

Comment:
@buremba driverUriParameters can be passed as "driver_uri_parameters" or "driverUriParameters" need tp be used?

Reply:
They can be passed as `driver_uri_parameters `